README.md

```diff
@@ -21,6 +21,8 @@ The following examples show how you can use the library with various Arduino com
 - [Arduino Ethernet Shield](https://github.com/256dpi/arduino-mqtt/blob/master/examples/ArduinoEthernetShield/ArduinoEthernetShield.ino)
 - [Arduino WiFi Shield](https://github.com/256dpi/arduino-mqtt/blob/master/examples/ArduinoWiFiShield/ArduinoWiFiShield.ino)
 - [Adafruit HUZZAH ESP8266](https://github.com/256dpi/arduino-mqtt/blob/master/examples/AdafruitHuzzahESP8266/AdafruitHuzzahESP8266.ino) ([SSL](https://github.com/256dpi/arduino-mqtt/blob/master/examples/AdafruitHuzzahESP8266_SSL/AdafruitHuzzahESP8266_SSL.ino))
+- [Arduino/Genuino WiFi101 Shield](https://www.arduino.cc/en/Main/ArduinoWiFiShield101)
```

Comment:
Would it be ok to link to the examples in the repository as the other links? You could add links to the product and other stuff in the example.